kolibri/auth/api.py

```diff
@@ -174,10 +174,18 @@ def get_queryset(self):
 
 class MembershipFilter(FilterSet):
     user_ids = CharFilter(method="filter_user_ids")
+    collection = CharFilter(method="filter_collection_id")
 
     def filter_user_ids(self, queryset, name, value):
         return queryset.filter(user_id__in=value.split(','))
 
+    def filter_collection_id(self, queryset, name, value):
+        # for deleting memberships, we want to include collections that belong to the target collection (ex. learnergroup belonging to classroom)
+        if self.request.method == 'DELETE':
```

Comment:
This seems like the wrong place to put this - couldn't this go on the delete method of the ViewSet or Serializer, rather than in the filter?